### ISSUE TITLE
[enhancement] score window and rank.

### ISSUE THREAD

**Issue by nieknooijens** [guusw/unnamed-sdvx-clone#15]
So after a song is finished at the moment it just freezes, you have to press escape to return to the song browser. It would be nice to have a screen that gives you a score evaluation with a rank (F E D C B A AA S). and then press <start> on the sdvx controller to go back to the song selection screen.

**Comment by guusw** [comment]
Ok, I'll create a milestone for basically all the things needed for the game to be played by someone. Then after that is done those things can be improved, etc.

**Comment by guusw** [comment]
![](http://tdrz.nl/JFS9eJA7.png])
[http://tdrz.nl/JFS9eJA7.png](http://tdrz.nl/JFS9eJA7.png)
Current design for the score screen (still missing max combo)

**Comment by nieknooijens** [comment]
Nice. Unfortunately we have an internet blackout here. Currently using my
phone.
So i can probably take a look tomorrow.

Op 3 sep. 2016 06:57 schreef "Guus" <notifications@github.com>:

**Comment by GhettoGirl** [comment]
I think the screen should also show the difficulty of the chart. Under the "Effected by" line seems a good place for it.

**Comment by GhettoGirl** [comment]
And maybe also show "Track failed", "Track completed", "Ultimate Chain", and "Perfect" somewhere. I kind of miss this string in the result screen. :smile:

**Comment by guusw** [comment]
Yep, I'll add the difficulty below the effecter, I assume you mean challenge,beginner,etc.
For the map result, maybe placing it as an overlay on the score graph would work, kind of like how osu! does it? (http://tdrz.nl/F6JKoBZH)
